supervision/detection/utils/vlms.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+
+def edit_distance(string_1: str, string_2: str, case_sensitive: bool = True) -> int:
```

Comment:
nit: `levenshtein_distance` would be a more suitable name for this function ([ref](https://en.wikipedia.org/wiki/Levenshtein_distance))